### ISSUE TITLE
Lighthouse ignores image source url query params

### ISSUE THREAD

**Issue by Atrix1987** [GoogleChrome/lighthouse#4100]
Lot of websites use external services like Imgix or ImageKit to optimize their own images. The configuration parameters are passed in the URL query params of the image source. Lighthouse ignores these parameters and shows the original image size and marks it an issues. While this might have been done for several reasons, it results in inaccurate test results.

**Comment by patrickhulce** [comment]
Thanks for filing @Atrix1987 do you have a public URL that demonstrates this issue?

Lighthouse should be reporting the natural size of whatever image ended up being downloaded, so this shouldn't be an issue. If you're seeing that behavior, it's a bug we need to investigate :)

**Comment by wardpeet** [comment]
@Atrix1987 could you send us an url to reproduce this issue?

**Comment by Atrix1987** [comment]
Hi @wardpeet here you go

**Public URL:** https://yourstory.com/2017/12/2017-language-tech-indian-internets-turning-point/
**Image Source URL:** https://images.yourstory.com/2017/12/Language.jpg?&fit=max&q=40&dpr=2
**Final Image Size** is 49KB
**Lighthouse** shows it as 198KB

<img width="718" alt="screen shot 2018-01-02 at 9 19 29 am" src="https://user-images.githubusercontent.com/2898596/34473960-eca16918-ef9e-11e7-9b16-8f556592a18e.png">

There are other images for which we are modifying the width and height using the URL params
**Sample Image URL**: https://images.yourstory.com/2018/01/Ross.jpg?auto=compress&h=120&w=120&fit=crop&crop=faces

Hope this helps.
Interestingly the same behaviour can be observed with [PageSpeed Insights Tool](https://developers.google.com/speed/pagespeed/insights/)

**Comment by patrickhulce** [comment]
tl;dr appears to work fine in latest version of LH

Thanks for providing the URL @Atrix1987! Tbh, I'm not really sure what's going on here. Lighthouse definitely doesn't strip any query params and is only examining images that were requested over the network.

When I audit that URL with LH 2.7 using the CLI or the extension, Chrome loads `https://images.yourstory.com/2017/12/Language.jpg?auto=compress` but when using 2.5.1 in DevTools I get the same result as you, Chrome loads `https://images.yourstory.com/2017/12/Language.jpg`. Really strange...

We'll double check when we push the latest version to DevTools that it isn't a bug with the integration there we need to look into.

**Comment by wardpeet** [comment]
@patrickhulce @Atrix1987 in latest canary everything works fine for me.

https://googlechrome.github.io/lighthouse/viewer/?gist=51dc5ff0804fc1bfe0f3b1b77d8886a7

**Comment by patrickhulce** [comment]
Awesome to hear thanks for double-checking @wardpeet! I'm going to go ahead and close this issue, but be aware that it might take awhile for Chrome 65 to make it to stable. You can use Canary/CLI/Extension in the meantime.

**Comment by Atrix1987** [comment]
Thanks a lot
